# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xf86-video-nsc/compile.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-nsc-2.8.3/src/amd_lx_shadow.c.vanilla	2008-10-21 16:04:56.000000000 +0200
+++ xf86-video-nsc-2.8.3/src/amd_lx_shadow.c	2008-10-21 16:04:59.000000000 +0200
@@ -48,7 +48,6 @@
 #include "xf86.h"
 #include "xf86_OSproc.h"
 #include "xf86Resources.h"
-#include "xf86_ansic.h"
 #include "xf86PciInfo.h"
 #include "xf86Pci.h"
 #include "nsc.h"
--- xf86-video-nsc-2.8.3/src/amd_lx_driver.c.vanilla	2008-10-21 16:03:22.000000000 +0200
+++ xf86-video-nsc-2.8.3/src/amd_lx_driver.c	2008-10-21 16:04:47.000000000 +0200
@@ -50,10 +50,10 @@
 /* Includes that are used by all drivers */
 #include "xf86.h"
 #include "xf86_OSproc.h"
-#include "xf86_ansic.h"
-#include "xf86_libc.h"
 #include "xf86Resources.h"
 
+#include <fcntl.h> /* RDONLY */
+
 /* We may want inb() and outb() */
 #include "compiler.h"
 
--- xf86-video-nsc-2.8.3/src/amd_lx_dga.c.vanilla	2008-10-21 16:03:14.000000000 +0200
+++ xf86-video-nsc-2.8.3/src/amd_lx_dga.c	2008-10-21 16:03:16.000000000 +0200
@@ -49,7 +49,6 @@
 
 #include "xf86.h"
 #include "xf86_OSproc.h"
-#include "xf86_ansic.h"
 #include "xf86Pci.h"
 #include "xf86PciInfo.h"
 #include "xaa.h"
--- xf86-video-nsc-2.8.3/src/amd_lx_video.c.vanilla	2008-10-21 16:05:07.000000000 +0200
+++ xf86-video-nsc-2.8.3/src/amd_lx_video.c	2008-10-21 16:05:09.000000000 +0200
@@ -51,7 +51,6 @@
 #include "xf86.h"
 #include "xf86_OSproc.h"
 #include "xf86Resources.h"
-#include "xf86_ansic.h"
 #include "compiler.h"
 #include "xf86PciInfo.h"
 #include "xf86Pci.h"
--- xf86-video-nsc-2.8.3/src/amd_lx_accel.c.vanilla	2008-10-21 16:02:49.000000000 +0200
+++ xf86-video-nsc-2.8.3/src/amd_lx_accel.c	2008-10-21 16:02:54.000000000 +0200
@@ -49,11 +49,9 @@
 
 #include "vgaHW.h"
 #include "xf86.h"
-#include "xf86_ansic.h"
 #include "xaalocal.h"
 #include "xf86fbman.h"
 #include "miline.h"
-#include "xf86_libc.h"
 #include "xaarop.h"
 #include "nsc.h"
 
--- xf86-video-nsc-2.8.3/src/amd_lx_cursor.c.vanilla	2008-10-21 16:03:04.000000000 +0200
+++ xf86-video-nsc-2.8.3/src/amd_lx_cursor.c	2008-10-21 16:03:06.000000000 +0200
@@ -48,7 +48,6 @@
 
 #include "xf86.h"
 #include "xf86_OSproc.h"
-#include "xf86_ansic.h"
 #include "xf86Pci.h"
 #include "xf86PciInfo.h"
 
